addons/mail/views/mail_templates.xml

```diff
@@ -81,6 +81,10 @@
                 <script type="text/javascript" src="/mail/static/src/js/views/activity/activity_record.js"></script>
                 <!-- Attachment box -->
                 <script type="text/javascript" src="/mail/static/src/js/attachment_box.js"></script>
+                <!-- Emojis -->
+                <script type="text/javascript" src="/mail/static/src/js/emojis_mixin.js"/>
+                <script type="text/javascript" src="/mail/static/src/js/text_emojis_field.js"/>
+                <link rel="stylesheet" type="text/scss" href="/mail/static/src/scss/emojis.scss"/>
```

Comment:
Fields a little strange that the `<!-- Emojis -->` section is just before

```
<!-- Others -->
<script type="text/javascript" src="/mail/static/src/js/emojis.js"></script>
```

Maybe re-group those?

Shouldn't emojis.js be before the new scripts as we use it (I think)?